src/main/plugin/iso19139.ca.HNAP/index-fields/language-default.xsl

```diff
@@ -378,6 +378,8 @@
         </xsl:when>
         <xsl:otherwise>
           <Field name="secConstr" string="false" store="true" index="true"/>
+          <xsl:variable name="securityConstraintsUseLimitation" select="if ($isoLangId = 'fra') then 'Inconnu' else 'Unknown'"/>
+          <Field name="MD_SecurityConstraintsUseLimitation" string="{$securityConstraintsUseLimitation}" store="true" index="true"/>
```

Comment:
If undefined the security constraining use limitation level is shown as `Unknown` / `Inconnu`

Reply:
Discussion: Prior systems probably assumed Unclassified, this makes it more specific as Unknown (for example from external systems).

Reply:
Aside: This is only for the user interface / editor display, the actual XML record is not fixed.

Looking at a future PR to address this during import.